byoqm/source_repository/source_repository.py

```diff
@@ -8,6 +8,10 @@
 
 
 class SourceRepository:
+    """
+    represents a repository containing all information accessible by metrics
```

Comment:
We are not using repository to refer to a code base, but rather to refer to the repository architecture.
```suggestion
    contains all information accessible by metrics
```